kustomize/commands/edit/add/configmap_test.go

```diff
@@ -26,7 +26,7 @@ const (
 
 func TestNewAddConfigMapIsNotNil(t *testing.T) {
 	fSys := filesys.MakeFsInMemory()
-	assert.NotNil(t, newCmdAddConfigMap(
+	require.NotNil(t, newCmdAddConfigMap(
```

Comment:
This there a reason we should use `assert` instead of `require`?

Reply:
[`require` fails the test right away](https://pkg.go.dev/github.com/stretchr/testify/require), instead of resuming test execution. I've been changing some usages where asserting other values might not be as useful once one of the assertions have failed already.

Reply:
This seems reasonable. Thanks for sharing.